src/wymeditor/editor/undo-redo.js

```diff
@@ -79,6 +79,9 @@ WYMeditor.UndoRedo.prototype._do = function (what) {
         if (history.changesetsBack.length === 0) {
             return;
         }
+        if (!history.changesetsFore.length) {
+            undoRedo._add()
```

Comment:
Style fixed.

> Also, the fact that we need to use an underscore method makes me think that either:

> * This should be handled in `history` via a public method
> * `_add()` should be `add()`

> if the only existing implementation of a library requires usage of private methods, that's a good sign we can improve the API.

Trying to understand what you're saying here it comes to my mind that perhaps you thought that `_add` is a method of an instance of object-history—that `undoRedo` here is an instance of object-history.

Is that true?

Otherwise, I didn't get where you're coming from.